src/plugins/intel_cpu/src/nodes/subgraph.h

```diff
@@ -107,15 +107,16 @@ class Snippet : public Node {
 
         private:
             static const size_t rank6D {6};
+            // Note: these are hint for domain optimization and tileRank selection
+            const size_t min_parallel_work_amount {16};
+            const size_t min_jit_work_amount {256};
```

Comment:
Could you tell me please if we really store this values as class fields? Or we can directly set these values?

Another my suggestion is below since `min_jit_work_amount` is static:
```suggestion
            const size_t min_parallel_work_amount {16};
            static constexpr size_t min_jit_work_amount {256};
```

Reply:
Right, we can set them directly, why not